Source/SwiftLintFramework/Rules/Lint/CommentSpacingRule.swift

```diff
@@ -0,0 +1,143 @@
+            Example("""
+            // Multiline double-slash
+            // comment
+            """),
+            Example("""
+            /// Multiline triple-slash
+            /// comment
+            """),
+            Example("""
+            /// Multiline triple-slash
+            ///   - This is indented
+            """),
+            Example("""
+            // - MARK: Mark comment
+            """),
+            Example("""
+            /* Asterisk comment */
+            """),
+            Example("""
+            /*
+                Multiline asterisk comment
+            */
+            """)
+        ],
+        triggeringExamples: [
+            Example("""
+            //↓Something
+            """),
+            Example("""
+            //↓MARK
+            """),
+            Example("""
+            //↓👨‍👨‍👦‍👦Something
+            """),
+            Example("""
+            func a() {
+                //↓This needs refactoring
+                print("Something")
+            }
+            //↓We should improve above function
+            """),
+            Example("""
+            ///↓This is a comment
+            """),
+            Example("""
+            /// Multiline triple-slash
+            ///↓This line is incorrect, though
+            """),
+            Example("""
+            //↓- MARK: Mark comment
+            """)
+        ],
+        corrections: [
+            Example("//↓Something"): Example("// Something"),
+            Example("//↓- MARK: Mark comment"): Example("// - MARK: Mark comment"),
+            Example("""
+            /// Multiline triple-slash
+            ///↓This line is incorrect, though
+            """): Example("""
+            /// Multiline triple-slash
+            /// This line is incorrect, though
+            """),
+            Example("""
+            func a() {
+                //↓This needs refactoring
+                print("Something")
+            }
+            //↓We should improve above function
+            """): Example("""
+            func a() {
+                // This needs refactoring
+                print("Something")
+            }
+            // We should improve above function
+            """)
+        ]
+    )
+
+    public func violationRanges(in file: SwiftLintFile) -> [NSRange] {
+        // Find all comment tokens in the file and regex search them for violations
+        let commentTokens = file.syntaxMap.tokens.filter { [.comment, .docComment].contains($0.kind) }
+        return commentTokens.compactMap { (token: SwiftLintSyntaxToken) -> [NSRange]? in
+            guard let commentBody = file.stringView.substringWithByteRange(token.range).map(StringView.init) else {
+                return nil
+            }
+            // Look for 2-3 slash characters followed immediately by a non-whitespace, non-slash
+            // character (this is a violation)
+            return regex("^(\\/){2,3}[^\\s\\/]").matches(in: commentBody, options: .anchored)
```

Comment:
This would read better as a raw string literal:

```suggestion
            return regex(#"^(\/){2,3}[^\s\/]"#).matches(in: commentBody, options: .anchored)
```